### ISSUE TITLE
No incoming call, only push notification

### ISSUE THREAD

**Issue by iBinh** [wireapp/wire-ios#5749]
I installed wire iOS from appstore, while trying to make audio/video call from app.wire.com, iOS app show only push notification `XXX is calling`, no incoming call display

Same behavior with own server deployment, I've set up iOS client, build release version, set up APNS, APNS_VOIP on aws SNS, but only push notification working, calls are not

**Comment by johnxnguyen** [comment]
@iBinh first thing to check is that `Settings > Options > Share with iOS` is turned on. Second thing to check is that you have granted microphone access permissions. The app will ask you for this permission if you try to make a call, or record a voice message. If these two things are satisfied, then you should be seeing the native call UI for an incoming call.

**Comment by iBinh** [comment]
- Checked all, but sometime it works, sometime doesn't.
- On-premise iOS version only show push notification

**Comment by johnxnguyen** [comment]
@iBinh what do you mean by On-premise iOS version? Also, would you be willing to send over some debug logs so I can see why the native call UI is not being displayed? You can do this by going to `Settings > Advanced > Send debug report`, then sending the prefilled email. You can of course inspect the contents of the **two** log files before sending.

**Comment by iBinh** [comment]
I mean iOS version with self-hosted wire-server 
This is 2 log file got from Wire
[previous.log](https://github.com/wireapp/wire-ios/files/9050945/previous.log)
[current.log](https://github.com/wireapp/wire-ios/files/9050946/current.log)

**Comment by iBinh** [comment]
Yes, and do you know why native call UI is not displayed on my self-hosted iOS version, only push notification received `XXX is calling` (Camera, microphone access granted)
